### PR TITLE
CVE-2019-14880 update

### PR DESCRIPTION
**Pull request by darunesh-RH**

### CODE REVIEW

File: 2019/14xxx/CVE-2019-14880.json [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        ]
+    },
+    "references": {
+        "reference_data": [
+            {
+                "url": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2019-14880",
+                "name": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2019-14880",
+                "refsource": "CONFIRM"
+            }
+        ]
```
Comment (iamleot) [inline 2019/14xxx/CVE-2019-14880.json]: Please add a reference to upstream security advisory:

 https://moodle.org/mod/forum/discuss.php?d=393583#p1586744

File: 2019/14xxx/CVE-2019-14880.json [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    },
+    "description": {
+        "description_data": [
+            {
+                "lang": "eng",
+                "value": "A vulnerability was found in Moodle 3.7 to 3.7.2, 3.6 to 3.6.6, 3.5 to 3.5.8 and earlier.  OAuth 2 providers who do not verify users' email address changes require additional verification during sign-up to reduce the risk of account compromise."            }
```
Comment (iamleot) [inline 2019/14xxx/CVE-2019-14880.json]: Please change the description to mark fixed version, i.e.:

 3.7.3, 3.6.7 and 3.5.9

**Comment by cve-team** [comment]
Just waiting on implementation of iamleot's comments Dhananjay and we'll get that merged. Thanks!

**Comment by cristina479** [comment]
Red Hat CNA,

Can you address the comments in this PR in order to publish this CVE ID to the CVE List?

Thank you,
Cristina

File: 2019/14xxx/CVE-2019-14880.json [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+                "url": "https://moodle.org/mod/forum/discuss.php?d=393583#p1586744"
+            }
+
+        ]
+    },
+    "description": {
+        "description_data": [
+            {
+                "lang": "eng",
+                "value": "A vulnerability was found in Moodle 3.7 to 3.7.2, 3.6 to 3.6.6, 3.5 to 3.5.8 and earlier.  OAuth 2 providers who do not verify users' email address changes require additional verification during sign-up to reduce the risk of account compromise."
```
Comment (iamleot) [inline 2019/14xxx/CVE-2019-14880.json]: Please change the sentence from:

 `[...] Moodle 3.7 to 3.7.2, 3.6 to 3.6.6, 3.5 to 3.5.8 and earlier [...]`

to:

  `[...] Moodle versions 3.7 before 3.7.3, 3.6 before 3.6.7, 3.5 before 3.5.9 and earlier [...]`

**Pull request merged**